packages/ckeditor5-image/src/imagestyle.js

```diff
@@ -352,50 +283,49 @@ export default class ImageStyle extends Plugin {
- * {@link module:image/imagestyle/imagestyleui~ImageStyleUI} plugin).
+ * The defined drop-down will be registered
+ * as the {@link module:ui/dropdown/dropdownview~DropdownView}
+ * with the {@link module:ui/dropdown/button/splitbuttonview~SplitButtonView} under the provided name in the
+ * {@link module:ui/componentfactory~ComponentFactory}
+ *
+ * @property {String} name The unique name of the drop-down. It is recommended to precede it with the "imageStyle:" prefix
+ * to avoid collision with the components' names registered by other plugins.
  *
- * @property {String} name The unique name of the group. It can refer to one of the
- * {@link module:image/imagestyle/utils~DEFAULT_GROUPS default groups} or be completely custom to define a new group.
- * It will be used for registering the drop-down  under the name `'imageStyle:{name}'`.
+ * @property {String} [title] The drop-down's title. It will be used as the split button label along with the title of the default item
+ * in the following manner: "Custom drop-down title: Default item title".
  *
- * @property {String} [title] The group's title. Setting `title` to one of
+ * Setting `title` to one of
  * {@link module:image/imagestyle/imagestyleui~ImageStyleUI#localizedDefaultStylesTitles}
  * will automatically translate it to the language of the editor.
  *
- * If this property is not defined, its value is inherited
- * from the {@link module:image/imagestyle/utils~DEFAULT_GROUPS default group} addressed in the name property.
- *
- * @property {Array.<String>} [items] The list of the names of the buttons that will be placed in the dropdown toolbar.
- * Each of the buttons has to be defined
- * as the {@link module:image/imagestyle~ImageStyleOptionDefinition image styling option} to be registered in the
- * {@link module:ui/componentfactory~ComponentFactory UI components factory}.
- *
- * If this property is not defined, its value is inherited
- * from the {@link module:image/imagestyle/utils~DEFAULT_GROUPS default group} addressed in the name property.
+ * @property {Array.<String>} items The list of the names of the buttons that will be placed in the drop-down's toolbar.
+ * Each of the buttons has to be one of the {@link module:image/image~ImageConfig#styles default image style buttons}
+ * or to be defined as the {@link module:image/imagestyle~ImageStyleOptionDefinition image styling option}.
  *
- * @property {String} [defaultItem] The name of one of the styling options from the items list,
+ * @property {String} defaultItem The name of one of the buttons from the items list,
  * which will be used as a default button for the drop-down's split button.
  *
- * If this property is not defined, its value is inherited
- * from the {@link module:image/imagestyle/utils~DEFAULT_GROUPS default group} addressed in the name property.
- *
- * @typedef {Object} module:image/imagestyle~ImageStyleGroupDefinition
+ * @typedef {Object} module:image/imagestyle~ImageStyleDropdownDefinition
```

Comment:
I guess this `typedef` could go to `ImageStyleUI`. It's purely presentational and decoupled from the editing. WDYT?